### ISSUE TITLE
Create a new S0 snapshot DB

### ISSUE THREAD

**Issue by sophoah** [harmony-one/harmony#4136]
Original snap DB created was only less than 50G, however as S0 DB is growing at 10G per day, it has already hit today 650G, It's time to plan for another snap.

**Comment by hedgeme** [comment]
DB is over 860GB, any word on when the new snap will be available?

**Comment by MaxMustermann2** [comment]
@sophoah can we close this issue now?

**Comment by hedgeme** [comment]
yes, this issue has since been resolved with the pruned DB. Thanks

**Comment by YuXiaoCoder** [comment]
如何修剪数据库呢？

**Comment by Frozen** [comment]
The reporter confirmed that the new pruned DB resolved the snapshot problem. Closing as completed.
